crates/re_viewer/src/ui/view_spatial/ui_3d.rs

```diff
@@ -238,18 +238,55 @@ fn find_camera(space_cameras: &[SpaceCamera3D], needle: &EntityPath) -> Option<E
 
 // ----------------------------------------------------------------------------
 
-pub const HELP_TEXT_3D: &str = "Drag to rotate.\n\
-    Drag with secondary mouse button to pan.\n\
-    Drag with middle mouse button (or primary mouse button + holding ALT/OPTION) to roll the view.\n\
-    Scroll to zoom.\n\
-    \n\
-    While hovering the 3D view, navigate with WSAD and QE.\n\
-    CTRL slows down, SHIFT speeds up.\n\
-    \n\
-    Double-click an object to focus the view on it.\n\
-    For cameras, you can restore the view again with Escape.\n\
-    \n\
-    Double-click on empty space to reset the view.";
+pub fn help_text(re_ui: &re_ui::ReUi) -> egui::WidgetText {
+    let pan_mouse = egui::PointerButton::Secondary;
+    let roll_mouse = egui::PointerButton::Middle;
+    let roll_mouse_alt = egui::PointerButton::Primary;
+    let roll_modifier = egui::Modifiers::ALT;
+    let speed_up = egui::Modifiers::SHIFT;
+    let slow_down = egui::Modifiers::CTRL;
+    let restore_key = egui::Key::Escape;
```

Comment:
Would be nice if these semantic constants were shared with the values used here:
https://github.com/rerun-io/rerun/blob/78c99361cb52cafe2c8894ab9ff9e58cef1763e5/crates/re_viewer/src/ui/view_spatial/eye.rs#L298-L317

Reply:
was too lazy to go that extra mile, but yeah at least some of those are pretty easy. Let's put them somewhere central!

Reply:
that would be nice

Reply:
done that now. Sadly lots of controls are hidden inside other egui constructs like scrollview or plot